CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [4.0.0] - 2024-05-21
+
+### **Breaking Changes**
+
+- Changed environment/config variables to use `AXIOS_DNS_CACHE` prefix
```

Comment:
none blocking, but did we update/add these variables in aws already?

Reply:
No I don't think we were using env variables to configure the cache - we were just using defaults

Reply:
right, good to know, thanks